train_file 文件夹已存在!
数据读取成功!
Traceback (most recent call last):
  File "C:\Users\Administrator\PycharmProjects\myproject\learn\trainer.py", line 181, in <module>
    start_train()
  File "C:\Users\Administrator\PycharmProjects\myproject\learn\trainer.py", line 178, in start_train
    total_timesteps = options.total_timesteps).train()
                                               ^^^^^^^
  File "C:\Users\Administrator\PycharmProjects\myproject\learn\trainer.py", line 81, in train
    env_train, env_trade = self.get_env(train_data, trade_data)
                           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Administrator\PycharmProjects\myproject\learn\trainer.py", line 127, in get_env
    folder=self.log_dir,  # 关键修复点
           ^^^^^^^^^^^^
AttributeError: 'Trainer' object has no attribute 'log_dir'
